packages/iris-grid/src/IrisGridPartitionSelector.tsx

```diff
@@ -205,13 +207,15 @@ class IrisGridPartitionSelector<T> extends Component<
         >
           Ignore &amp; Fetch All
         </button>
-        <button
-          type="button"
-          className="btn btn-outline-primary btn-append"
-          onClick={this.handleAppendClick}
-        >
-          Append Command
-        </button>
+        {onAppend === undefined || (
```

Comment:
I'd prefer to write this as the truthy case and use `&&` instead of the falsey case and use `||`, just for consistency with what we do elsewhere in the app, e.g:
```suggestion
        {onAppend !== undefined && (
```